licenses/git_utils.py

```diff
@@ -0,0 +1,41 @@
+import git
+
+
+def setup_local_branch(repo: git.Repo, branch_name: str, parent_branch_name: str):
+    """
+    Ensure we have a local branch named 'branch_name'.
+    Pull from upstream ('origin'), or create as a branch from parent_branch_name.
+    If there's an upstream one, pull from it to make sure we're up to date.
+    Check it out.
+    """
+    # is there already a branch?
+    origin = repo.remotes.origin
+    origin.fetch()
+    if not hasattr(repo.branches, branch_name):
+        # Not locally, maybe upstream
+        if hasattr(origin.refs, branch_name):
+            repo.create_head(branch_name, f"origin/{branch_name}")
+        else:
+            # Nope, need to create from scratch
+            print("create from scratch")
+            parent_branch = getattr(origin.refs, parent_branch_name)
+            repo.create_head(branch_name, parent_branch)
+    else:
+        print("local already exists")
+    branch = getattr(repo.heads, branch_name)
+    branch.checkout()
+    # Make sure local branch is up to date if there's an upstream branch
+    if hasattr(origin.refs, branch_name):
+        origin.pull(f"{branch_name}:{branch_name}")
+
+
+def commit_and_push_changes(repo: git.Repo, branch_name: str, commit_msg: str):
+    """Commit new translation changes to current branch, and push upstream"""
```

Comment:
It looks like we can commit to any branch through this definition. Can we remove `"current"` from the docstring.

Reply:
Good point - it's actually committing the changes to the current branch, then pushing whatever branch was passed in, which doesn't make a lot of sense. I'll change it to just work on the current branch without passing in a name.